jsBasics/homeWork5/task1.js

```diff
@@ -0,0 +1,22 @@
+//function declaration
+function rectangleAreaDeclaration(width, height){
+    let result = width * height;
+    return result;
+}
+console.log("The area of the rectangle :", rectangleAreaDeclaration(23, 16));
+//----------------------------------------------------------------------------------------------------------------------
+
+//function expression
+const rectangleAreaExpression = function (width, height) {
+    let result = width * height;
+    return result;
+}
+console.log("The area of the rectangle :",rectangleAreaExpression(26, 32));
+//----------------------------------------------------------------------------------------------------------------------
+
+//arrow function expression
+const rectangleAreaArrow = (width, height) => {
+    let result = width * height;
+    return result;
+}
```

Comment:
why let but not const?

Reply:
function name should represent and action

Reply:
@sttaran I didn't find in our lesson that I can't use let in functions. But if points are deducted for this, then I corrected it.